docs/docs/start/docker_dev.md

```diff
@@ -107,6 +99,17 @@ Now that the database has been created, migrations applied, and you have created
 docker compose up -d
 ```
 
+### Create Admin Account
+
+If you are creating the initial database, you need to create an admin (superuser) account for the database. Run the command below, and follow the prompts:
+
+!!! info "Containers must be running"
```

Comment:
Doesn't dc Handels this for you if the containers are marked as depends_on?

Reply:
I've been having a lot of issues getting a superuser if I run `invoke superuser` before the initial `docker compose up -d`
The command works perfectly after the `up` command, but is very flimsy before.

Reply:
Oh yeah, could be that the db is not yet ready if they are started just before the invoke superuser

Reply:
That's also the reason I moved the superuser command down below the up command. That way, the db will def be running before, if you follow the doc sequentially.